frontend/src/metabase/query_builder/components/expressions/ExpressionEditorSuggestions/ExpressionEditorSuggestions.unit.spec.tsx

```diff
@@ -83,4 +86,47 @@ describe("ExpressionEditorSuggestions", () => {
       1,
     );
   });
+
+  it("should render correct functions link for expressions", async () => {
+    setup({ startRule: "expression" });
+    expect(screen.getByText("View all functions")).toBeInTheDocument();
+    expect(
+      screen
+        .getByText("View all functions")
+        .getAttribute("href")
+        ?.endsWith("#functions"),
+    ).toBe(true);
+  });
+
+  it("should render correct functions link for filters", async () => {
+    setup({ startRule: "boolean" });
+    expect(screen.getByText("View all functions")).toBeInTheDocument();
+    expect(
+      screen
+        .getByText("View all functions")
+        .getAttribute("href")
+        ?.endsWith("#functions"),
+    ).toBe(true);
+  });
+
+  it("should render correct functions link for aggregations", async () => {
+    setup({ startRule: "aggregation" });
+    expect(screen.getByText("View all functions")).toBeInTheDocument();
+    expect(
+      screen
+        .getByText("View all functions")
+        .getAttribute("href")
+        ?.endsWith("#aggregations"),
+    ).toBe(true);
+  });
+
+  it("should  functions when first opened", async () => {
```

Comment:
```suggestion
  it("should [verb] functions when first opened", async () => {
```

Also: redundant `async` (multiple tests in this file)